src/js/apps/patients/sidebar/filters-sidebar_app.js

```diff
@@ -39,10 +55,35 @@ export default App.extend({
 
     this.showChildView('customFilters', customFiltersView);
   },
+  showStatesFiltersView() {
+    const statesFiltersView = new StatesFiltersView({
+      collection: this.defaultStatesFilter,
+      model: this.getState(),
+    });
+
+    this.showChildView('statesFilters', statesFiltersView);
+  },
   onClearFilters() {
-    this.getState().clear();
+    const defaultStates = this.defaultStatesFilter.map('id');
+
+    this.getState().clear().set({ states: defaultStates });
   },
   onStop() {
     Radio.request('sidebar', 'close');
   },
+  getFiltersCount() {
```

Comment:
I think the way we can reduce this duplication is to make a shared state model that this function lives on like the `worklist_state.js` that both of these filter_apps use.  Then we could also go back to passing the state model as the model to the button and header views rather than re-rendering them in the parent.